thirdeye/thirdeye-frontend/app/pods/components/filter-bar/template.hbs

```diff
@@ -0,0 +1,28 @@
+{{!-- TODO: Create separate styling for filter-bar component --}}
+<div class="entity-filter filter-bar">
+
+  {{#each config as |block|}}
+    <section class="entity-filter__group">
+      <h5 class="entity-filter__group-title">{{block.header}}
+        <i class="glyphicon"></i>
```

Comment:
is this a placeholder?

Reply:
Yes, it is. A lot of this will be changed in later PRs.